core/src/main/java/nl/nn/adapterframework/pipes/MessageSendingPipe.java

```diff
@@ -362,14 +360,24 @@ public void configure() throws ConfigurationException {
 			}
 		}
 		ITransactionalStorage messageLog = getMessageLog();
-		if (checkMessageLog) {
-			if (!getSender().isSynchronous() && getListener()==null && !(getSender() instanceof nl.nn.adapterframework.senders.IbisLocalSender)) {
-				if (messageLog==null) {
-					ConfigurationWarnings.add(this, log, "asynchronous sender [" + getSender().getName() + "] without sibling listener has no messageLog. Integrity check not possible");
+		if (messageLog==null) {
+			if (!getSender().isSynchronous() && getListener()==null && !(getSender() instanceof nl.nn.adapterframework.senders.IbisLocalSender)) { // sender is asynchronous and not a local sender, but has no messageLog
+				boolean suppressIntegrityCheckWarning = ConfigurationWarnings.isSuppressed("warnings.suppress.integrityCheck", getAdapter(), getConfigurationClassLoader());
+				if (!suppressIntegrityCheckWarning) {
+					boolean legacyCheckMessageLog = AppConstants.getInstance(getConfigurationClassLoader()).getBoolean("messageLog.check", true);
+					if (!legacyCheckMessageLog) {
+						ConfigurationWarnings.add(this, log, "Suppressing integrityCheck warnings by setting property 'messageLog.check=false' has been replaced by by setting property 'warnings.suppress.integrityCheck=true'");
+						suppressIntegrityCheckWarning=true;
+					}
```

Comment:
Misschien in de configure zetten en zorgen dat dit maar 1 maal per configuratie voor kan komen?